src/app/components/ContractVerificationIcon/index.tsx

```diff
@@ -87,15 +90,32 @@ export const VerificationIcon: FC<{ address_eth: string; verified: boolean; noLi
     verified: t('contract.verification.isVerified'),
     unverified: t('contract.verification.isNotVerified'),
   }
-  const linkProps = {
+  const sourcifyLinkProps = {
     href: `https://sourcify.dev/#/lookup/${address_eth}`,
     rel: 'noopener noreferrer',
     target: '_blank',
     sx: { fontWeight: 400, color: 'inherit', textDecoration: 'underline' },
     onClick: verified ? undefined : () => setExplainDelay(true),
   }
   const Component = noLink ? Box : Link
-  const componentProps = noLink ? {} : linkProps
+  const componentProps = noLink ? {} : sourcifyLinkProps
+
+  const scopeToPlaygroundURL: Record<Network, Partial<Record<Layer, string>>> = {
+    [Network.mainnet]: {
+      [Layer.emerald]: `https://abi-playground.oasis.io/?network=42262&contractAddress=${address_eth}`,
+      [Layer.sapphire]: `https://abi-playground.oasis.io/?network=23294&contractAddress=${address_eth}`,
+    },
+    [Network.testnet]: {
+      [Layer.emerald]: `https://abi-playground.oasis.io/?network=42261&contractAddress=${address_eth}`,
+      [Layer.sapphire]: `https://abi-playground.oasis.io/?network=23295&contractAddress=${address_eth}`,
+    },
+  }
```

Comment:
Can this be moved to some kind of config? ABI playground base url would also be nice to be extracted, so it's easy configurable.